src/main/resources/model/instance.json

```diff
@@ -91,6 +91,11 @@
         "contributorTypeId": {
           "index": "keyword",
           "showInResponse": [ "search", "call-number-browse" ]
+        },
+        "authorityId": {
+          "index": "keyword",
+          "searchAliases": [ "authorityId" ],
+          "showInResponse": [ "search", "call-number-browse" ]
         }
```

Comment:
Please update the documentation with new search options

Reply:
done